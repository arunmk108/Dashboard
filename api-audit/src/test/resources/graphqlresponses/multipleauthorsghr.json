{
    "data": {
        "repository": {
            "pullRequests": {
                "totalCount": 2,
                "pageInfo": {
                    "endCursor": "Y3Vyc29yOnYyOpK5MjAxOC0wMS0yNlQyMzowMzozMC0wNTowMM4ADjvp",
                    "hasNextPage": false
                },
                "edges": [
                    {
                        "cursor": "Y3Vyc29yOnYyOpK5MjAxOC0wMS0yNlQyMzoxNDo1Ni0wNTowMM4ADjvq",
                        "node": {
                            "number": 2,
                            "mergeable": "UNKNOWN",
                            "state": "MERGED",
                            "createdAt": "2018-01-27T04:05:51Z",
                            "updatedAt": "2018-01-27T04:14:56Z",
                            "mergedAt": "2018-01-27T04:14:56Z",
                            "baseRef": {
                                "target": {
                                    "oid": "b957f24c0506d96042a5bb698d0933184af7bcc3"
                                }
                            },
                            "baseRefName": "master",
                            "title": "second pr from dev to master",
                            "mergeCommit": {
                                "oid": "b957f24c0506d96042a5bb698d0933184af7bcc3"
                            },
                            "headRepository": {
                                "name": "MultipleAuthorsGHR",
                                "url": "https://mygithub.com/Devopscode/MultipleAuthorsGHR"
                            },
                            "headRef": {
                                "target": {
                                    "oid": "27ceabaca1453809410bb0a40d14a201a5076c5b"
                                }
                            },
                            "headRefName": "dev",
                            "author": {
                                "login": "ghone",
                                "resourcePath": "/ghone"
                            },
                            "commits": {
                                "totalCount": 3,
                                "nodes": [
                                    {
                                        "commit": {
                                            "oid": "46c3bc434c935df51a350439d0faae0fd03a5e64",
                                            "committedDate": "2018-01-27T04:05:13Z",
                                            "message": "Create Anotherfile.txt",
                                            "status": null,
                                            "author": {
                                                "name": "Author One",
                                                "date": "2018-01-26T23:05:13-05:00",
                                                "user": {
                                                    "login": "ghone"
                                                }
                                            }
                                        }
                                    },
                                    {
                                        "commit": {
                                            "oid": "2f98b1cc6cfcf47e2b69a642d73c7e6c8ca1962a",
                                            "committedDate": "2018-01-27T04:05:26Z",
                                            "message": "Update Anotherfile.txt",
                                            "status": null,
                                            "author": {
                                                "name": "Author One",
                                                "date": "2018-01-26T23:05:26-05:00",
                                                "user": {
                                                    "login": "ghone"
                                                }
                                            }
                                        }
                                    },
                                    {
                                        "commit": {
                                            "oid": "27ceabaca1453809410bb0a40d14a201a5076c5b",
                                            "committedDate": "2018-01-27T04:08:53Z",
                                            "message": "Update Anotherfile.txt",
                                            "status": null,
                                            "author": {
                                                "name": "Author Two",
                                                "date": "2018-01-26T23:08:53-05:00",
                                                "user": {
                                                    "login": "ghtwo"
                                                }
                                            }
                                        }
                                    }
                                ]
                            },
                            "comments": {
                                "totalCount": 1,
                                "nodes": [
                                    {
                                        "bodyText": "LGTM whose code ?",
                                        "author": {
                                            "login": "ghone"
                                        },
                                        "createdAt": "2018-01-27T04:13:56Z",
                                        "updatedAt": "2018-01-27T04:13:56Z"
                                    }
                                ]
                            },
                            "reviews": {
                                "totalCount": 1,
                                "nodes": [
                                    {
                                        "id": "MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NTA3NDI4",
                                        "bodyText": "Yes! Approving my own change!",
                                        "state": "APPROVED",
                                        "author": {
                                            "login": "ghtwo"
                                        },
                                        "createdAt": "2018-01-27T04:10:33Z",
                                        "updatedAt": "2018-01-27T04:10:33Z"
                                    }
                                ]
                            }
                        }
                    },
                    {
                        "cursor": "Y3Vyc29yOnYyOpK5MjAxOC0wMS0yNlQyMzowMzozMC0wNTowMM4ADjvp",
                        "node": {
                            "number": 1,
                            "mergeable": "UNKNOWN",
                            "state": "MERGED",
                            "createdAt": "2018-01-27T03:56:36Z",
                            "updatedAt": "2018-01-27T04:03:30Z",
                            "mergedAt": "2018-01-27T04:03:17Z",
                            "baseRef": {
                                "target": {
                                    "oid": "b957f24c0506d96042a5bb698d0933184af7bcc3"
                                }
                            },
                            "baseRefName": "master",
                            "title": "pr from dev to master",
                            "mergeCommit": {
                                "oid": "d24f42ee7c706fefcc7c71e49f39e686f0d87f77"
                            },
                            "headRepository": {
                                "name": "MultipleAuthorsGHR",
                                "url": "https://mygithub.com/Devopscode/MultipleAuthorsGHR"
                            },
                            "headRef": {
                                "target": {
                                    "oid": "27ceabaca1453809410bb0a40d14a201a5076c5b"
                                }
                            },
                            "headRefName": "dev",
                            "author": {
                                "login": "ghone",
                                "resourcePath": "/ghone"
                            },
                            "commits": {
                                "totalCount": 3,
                                "nodes": [
                                    {
                                        "commit": {
                                            "oid": "88b1e10723154f0ff0ae942e5243c225cf95f235",
                                            "committedDate": "2018-01-27T03:55:49Z",
                                            "message": "Create somefile.txt",
                                            "status": null,
                                            "author": {
                                                "name": "Author One",
                                                "date": "2018-01-26T22:55:49-05:00",
                                                "user": {
                                                    "login": "ghone"
                                                }
                                            }
                                        }
                                    },
                                    {
                                        "commit": {
                                            "oid": "1ffbaeb80b6f90e84d5bd1ceb9d8d256202becb7",
                                            "committedDate": "2018-01-27T03:56:05Z",
                                            "message": "Update somefile.txt",
                                            "status": null,
                                            "author": {
                                                "name": "Author One",
                                                "date": "2018-01-26T22:56:05-05:00",
                                                "user": {
                                                    "login": "ghone"
                                                }
                                            }
                                        }
                                    },
                                    {
                                        "commit": {
                                            "oid": "d8bb61c9fb5342a6488f2796dcf4ded2e056d6ae",
                                            "committedDate": "2018-01-27T04:00:29Z",
                                            "message": "Update README.md",
                                            "status": null,
                                            "author": {
                                                "name": "Author Two",
                                                "date": "2018-01-26T23:00:29-05:00",
                                                "user": {
                                                    "login": "ghtwo"
                                                }
                                            }
                                        }
                                    }
                                ]
                            },
                            "comments": {
                                "totalCount": 0,
                                "nodes": []
                            },
                            "reviews": {
                                "totalCount": 1,
                                "nodes": [
                                    {
                                        "id": "MDE3OlB1bGxSZXF1ZXN0UmV2aWV3NTA3NDI3",
                                        "bodyText": "Good stuff.",
                                        "state": "APPROVED",
                                        "author": {
                                            "login": "ghtwo"
                                        },
                                        "createdAt": "2018-01-27T04:02:03Z",
                                        "updatedAt": "2018-01-27T04:02:03Z"
                                    }
                                ]
                            }
                        }
                    }
                ]
            },
            "ref": {
                "target": {
                    "history": {
                        "pageInfo": {
                            "endCursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 8",
                            "hasNextPage": false
                        },
                        "edges": [
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 0",
                                "node": {
                                    "oid": "b957f24c0506d96042a5bb698d0933184af7bcc3",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "d24f42ee7c706fefcc7c71e49f39e686f0d87f77"
                                            },
                                            {
                                                "oid": "27ceabaca1453809410bb0a40d14a201a5076c5b"
                                            }
                                        ]
                                    },
                                    "message": "Merge pull request #2 from Devopscode/dev\n\nsecond pr from dev to master",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:14:55-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T23:14:55-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 1",
                                "node": {
                                    "oid": "27ceabaca1453809410bb0a40d14a201a5076c5b",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "2f98b1cc6cfcf47e2b69a642d73c7e6c8ca1962a"
                                            }
                                        ]
                                    },
                                    "message": "Update Anotherfile.txt",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:08:53-05:00"
                                    },
                                    "author": {
                                        "name": "Author Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2018-01-26T23:08:53-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 2",
                                "node": {
                                    "oid": "2f98b1cc6cfcf47e2b69a642d73c7e6c8ca1962a",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "46c3bc434c935df51a350439d0faae0fd03a5e64"
                                            }
                                        ]
                                    },
                                    "message": "Update Anotherfile.txt",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:05:26-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T23:05:26-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 3",
                                "node": {
                                    "oid": "46c3bc434c935df51a350439d0faae0fd03a5e64",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "d24f42ee7c706fefcc7c71e49f39e686f0d87f77"
                                            }
                                        ]
                                    },
                                    "message": "Create Anotherfile.txt",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:05:13-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T23:05:13-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 4",
                                "node": {
                                    "oid": "d24f42ee7c706fefcc7c71e49f39e686f0d87f77",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "1f1238084447b81d6cbf3ecc02d307f98289b03d"
                                            },
                                            {
                                                "oid": "d8bb61c9fb5342a6488f2796dcf4ded2e056d6ae"
                                            }
                                        ]
                                    },
                                    "message": "Merge pull request #1 from Devopscode/dev\n\npr from dev to master",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:03:15-05:00"
                                    },
                                    "author": {
                                        "name": "Author Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2018-01-26T23:03:15-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 5",
                                "node": {
                                    "oid": "d8bb61c9fb5342a6488f2796dcf4ded2e056d6ae",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "1ffbaeb80b6f90e84d5bd1ceb9d8d256202becb7"
                                            }
                                        ]
                                    },
                                    "message": "Update README.md",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T23:00:29-05:00"
                                    },
                                    "author": {
                                        "name": "Author Two",
                                        "user": {
                                            "login": "ghtwo"
                                        },
                                        "email": "Author.Two@mycompany.com",
                                        "date": "2018-01-26T23:00:29-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 6",
                                "node": {
                                    "oid": "1ffbaeb80b6f90e84d5bd1ceb9d8d256202becb7",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "88b1e10723154f0ff0ae942e5243c225cf95f235"
                                            }
                                        ]
                                    },
                                    "message": "Update somefile.txt",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T22:56:05-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T22:56:05-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 7",
                                "node": {
                                    "oid": "88b1e10723154f0ff0ae942e5243c225cf95f235",
                                    "parents": {
                                        "nodes": [
                                            {
                                                "oid": "1f1238084447b81d6cbf3ecc02d307f98289b03d"
                                            }
                                        ]
                                    },
                                    "message": "Create somefile.txt",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T22:55:49-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T22:55:49-05:00"
                                    },
                                    "status": null
                                }
                            },
                            {
                                "cursor": "b957f24c0506d96042a5bb698d0933184af7bcc3 8",
                                "node": {
                                    "oid": "1f1238084447b81d6cbf3ecc02d307f98289b03d",
                                    "parents": {
                                        "nodes": []
                                    },
                                    "message": "Initial commit",
                                    "committer": {
                                        "user": null,
                                        "name": "GitHub Enterprise",
                                        "date": "2018-01-26T22:54:20-05:00"
                                    },
                                    "author": {
                                        "name": "Author One",
                                        "user": {
                                            "login": "ghone"
                                        },
                                        "email": "Author.One@mycompany.com",
                                        "date": "2018-01-26T22:54:20-05:00"
                                    },
                                    "status": null
                                }
                            }
                        ]
                    }
                }
            },
            "issues": {
                "totalCount": 0,
                "pageInfo": {
                    "endCursor": null,
                    "hasNextPage": false
                },
                "edges": []
            }
        }
    }
}