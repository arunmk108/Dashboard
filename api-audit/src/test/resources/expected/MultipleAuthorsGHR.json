[
    {
        "auditStatuses": [
            "PEER_REVIEW_GHR",
            "COMMITAUTHOR_EQ_MERGECOMMITER",
            "PULLREQ_REVIEWED_BY_PEER",
            "GIT_BRANCH_STRATEGY"
        ],
        "lastUpdated": 1517029042023,
        "pullRequest": {
            "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
            "scmBranch": "master",
            "scmRevisionNumber": "b957f24c0506d96042a5bb698d0933184af7bcc3",
            "scmCommitLog": "second pr from dev to master",
            "scmCommitTimestamp": 1517026496000,
            "numberOfChanges": 0,
            "id": "5a6c06b2f5fb7848d2820e57",
            "orgName": "Devopscode",
            "repoName": "MultipleAuthorsGHR",
            "sourceRepo": "Devopscode/MultipleAuthorsGHR",
            "sourceBranch": "dev",
            "targetRepo": "Devopscode/MultipleAuthorsGHR",
            "targetBranch": "master",
            "number": "2",
            "collectorItemId": "5a4e524333ec296b81c3935f",
            "updatedAt": 1517026496000,
            "createdAt": 1517025951000,
            "closedAt": 0,
            "state": "merged",
            "mergedAt": 1517026496000,
            "timestamp": 1517029041962,
            "resolutiontime": 0,
            "userId": "ghone",
            "comments": [
                {
                    "user": "ghone",
                    "createdAt": 1517026436000,
                    "updatedAt": 1517026436000,
                    "body": "LGTM whose code ?",
                    "status": ""
                }
            ],
            "reviews": [
                {
                    "body": "Yes! Approving my own change!",
                    "state": "APPROVED",
                    "author": "ghtwo",
                    "createdAt": 1517026233000,
                    "updatedAt": 1517026233000
                }
            ],
            "headSha": "27ceabaca1453809410bb0a40d14a201a5076c5b",
            "baseSha": "b957f24c0506d96042a5bb698d0933184af7bcc3",
            "requestType": "pull",
            "commits": [
                {
                    "scmRevisionNumber": "46c3bc434c935df51a350439d0faae0fd03a5e64",
                    "scmCommitLog": "Create Anotherfile.txt",
                    "scmAuthor": "Author One",
                    "scmAuthorLogin": "ghone",
                    "scmCommitTimestamp": 1517025913000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                },
                {
                    "scmRevisionNumber": "2f98b1cc6cfcf47e2b69a642d73c7e6c8ca1962a",
                    "scmCommitLog": "Update Anotherfile.txt",
                    "scmAuthor": "Author One",
                    "scmAuthorLogin": "ghone",
                    "scmCommitTimestamp": 1517025926000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                },
                {
                    "scmRevisionNumber": "27ceabaca1453809410bb0a40d14a201a5076c5b",
                    "scmCommitLog": "Update Anotherfile.txt",
                    "scmAuthor": "Author Two",
                    "scmAuthorLogin": "ghtwo",
                    "scmCommitTimestamp": 1517026133000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                }
            ]
        },
        "commits": [
            {
                "scmRevisionNumber": "46c3bc434c935df51a350439d0faae0fd03a5e64",
                "scmCommitLog": "Create Anotherfile.txt",
                "scmAuthor": "Author One",
                "scmAuthorLogin": "ghone",
                "scmCommitTimestamp": 1517025913000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            },
            {
                "scmRevisionNumber": "2f98b1cc6cfcf47e2b69a642d73c7e6c8ca1962a",
                "scmCommitLog": "Update Anotherfile.txt",
                "scmAuthor": "Author One",
                "scmAuthorLogin": "ghone",
                "scmCommitTimestamp": 1517025926000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            },
            {
                "scmRevisionNumber": "27ceabaca1453809410bb0a40d14a201a5076c5b",
                "scmCommitLog": "Update Anotherfile.txt",
                "scmAuthor": "Author Two",
                "scmAuthorLogin": "ghtwo",
                "scmCommitTimestamp": 1517026133000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            }
        ],
        "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
        "scmBranch": "master"
    },
    {
        "auditStatuses": [
            "PEER_REVIEW_GHR",
            "COMMITAUTHOR_NE_MERGECOMMITER",
            "PULLREQ_NOT_PEER_REVIEWED",
            "PEER_REVIEW_GHR_SELF_APPROVAL",
            "GIT_BRANCH_STRATEGY"
        ],
        "lastUpdated": 1517029042023,
        "pullRequest": {
            "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
            "scmBranch": "master",
            "scmRevisionNumber": "d24f42ee7c706fefcc7c71e49f39e686f0d87f77",
            "scmCommitLog": "pr from dev to master",
            "scmCommitTimestamp": 1517025797000,
            "numberOfChanges": 0,
            "id": "5a6c06b2f5fb7848d2820e58",
            "orgName": "Devopscode",
            "repoName": "MultipleAuthorsGHR",
            "sourceRepo": "Devopscode/MultipleAuthorsGHR",
            "sourceBranch": "dev",
            "targetRepo": "Devopscode/MultipleAuthorsGHR",
            "targetBranch": "master",
            "number": "1",
            "collectorItemId": "5a4e524333ec296b81c3935f",
            "updatedAt": 1517025810000,
            "createdAt": 1517025396000,
            "closedAt": 0,
            "state": "merged",
            "mergedAt": 1517025797000,
            "timestamp": 1517029041967,
            "resolutiontime": 0,
            "userId": "ghone",
            "comments": [],
            "reviews": [
                {
                    "body": "Good stuff.",
                    "state": "APPROVED",
                    "author": "ghtwo",
                    "createdAt": 1517025723000,
                    "updatedAt": 1517025723000
                }
            ],
            "headSha": "27ceabaca1453809410bb0a40d14a201a5076c5b",
            "baseSha": "b957f24c0506d96042a5bb698d0933184af7bcc3",
            "requestType": "pull",
            "commits": [
                {
                    "scmRevisionNumber": "88b1e10723154f0ff0ae942e5243c225cf95f235",
                    "scmCommitLog": "Create somefile.txt",
                    "scmAuthor": "Author One",
                    "scmAuthorLogin": "ghone",
                    "scmCommitTimestamp": 1517025349000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                },
                {
                    "scmRevisionNumber": "1ffbaeb80b6f90e84d5bd1ceb9d8d256202becb7",
                    "scmCommitLog": "Update somefile.txt",
                    "scmAuthor": "Author One",
                    "scmAuthorLogin": "ghone",
                    "scmCommitTimestamp": 1517025365000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                },
                {
                    "scmRevisionNumber": "d8bb61c9fb5342a6488f2796dcf4ded2e056d6ae",
                    "scmCommitLog": "Update README.md",
                    "scmAuthor": "Author Two",
                    "scmAuthorLogin": "ghtwo",
                    "scmCommitTimestamp": 1517025629000,
                    "numberOfChanges": 0,
                    "timestamp": 0,
                    "firstEverCommit": false
                }
            ]
        },
        "commits": [
            {
                "scmRevisionNumber": "88b1e10723154f0ff0ae942e5243c225cf95f235",
                "scmCommitLog": "Create somefile.txt",
                "scmAuthor": "Author One",
                "scmAuthorLogin": "ghone",
                "scmCommitTimestamp": 1517025349000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            },
            {
                "scmRevisionNumber": "1ffbaeb80b6f90e84d5bd1ceb9d8d256202becb7",
                "scmCommitLog": "Update somefile.txt",
                "scmAuthor": "Author One",
                "scmAuthorLogin": "ghone",
                "scmCommitTimestamp": 1517025365000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            },
            {
                "scmRevisionNumber": "d8bb61c9fb5342a6488f2796dcf4ded2e056d6ae",
                "scmCommitLog": "Update README.md",
                "scmAuthor": "Author Two",
                "scmAuthorLogin": "ghtwo",
                "scmCommitTimestamp": 1517025629000,
                "numberOfChanges": 0,
                "timestamp": 0,
                "firstEverCommit": false
            }
        ],
        "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
        "scmBranch": "master"
    },
    {
        "auditStatuses": [
            "DIRECT_COMMITS_TO_BASE_FIRST_COMMIT",
            "SCM_AUTHOR_LOGIN_INVALID"
        ],
        "lastUpdated": 1517029042023,
        "commits": [
            {
                "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
                "scmBranch": "master",
                "scmRevisionNumber": "1f1238084447b81d6cbf3ecc02d307f98289b03d",
                "scmCommitLog": "Initial commit",
                "scmAuthor": "Author One",
                "scmAuthorLogin": "ghone",
                "scmParentRevisionNumbers": [],
                "scmCommitTimestamp": 1517025260000,
                "numberOfChanges": 1,
                "type": "New",
                "id": "5a6c06b1f5fb7848d2820e56",
                "collectorItemId": "5a4e524333ec296b81c3935f",
                "timestamp": 1517029041968,
                "firstEverCommit": true
            }
        ],
        "scmUrl": "https://mygithub.com/Devopscode/MultipleAuthorsGHR",
        "scmBranch": "master"
    }
]